core/components/molecules/inputMask/__tests__/InputMask.test.tsx

```diff
@@ -125,4 +125,15 @@ describe('Input Mask component prop: onClearHandler', () => {
     fireEvent.click(input);
     expect(onClearHandler).toHaveBeenCalledTimes(1);
   });
+
+  it('with onClear icon invisile when value equals default placeholder', () => {
+    const value = '1111 2222 3333 4444';
+    const { getByTestId } = render(
+      <InputMask mask={mask} onClear={onClearHandler} value={value} placeholder="____ ____ ____ ____" />
+    );
+    const onClearIcon = getByTestId('DesignSystem-Input--closeIcon');
+    expect(onClearIcon).toBeInTheDocument();
+    fireEvent.click(onClearIcon);
```

Comment:
We can also check if `onClearHandler` function is called on close icon click.

Reply:
this usecase has alreday been covered by another prior test case.